packages/components/src/components/auto-height-wrapper/auto-height-wrapper.tsx

```diff
@@ -1,8 +1,18 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import { usePrevious } from '../../hooks';
 
-const AutoHeightWrapper = props => {
+export type TAutoHeightWrapperChildProps = {
```

Comment:
why do we need to export this type definition?

Reply:
We don't need to, I'll remove the export. Thank you